estate_account/models/estate_property.py

```diff
@@ -0,0 +1,43 @@
+from odoo import models, Command
+from odoo.exceptions import UserError
+
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def _get_default_journal(self):
+        return self.env['account.journal'].search([
+            *self.env['account.journal']._check_company_domain(self.env.company),
+            ('type', '=', 'sale'),
+        ], limit=1)
+
+    def _prepare_invoice_lines(self):
+        return [
+                    Command.create({
+                        'name': 'Selling price - 6%',
+                        'quantity': '1',
+                        'price_unit': self.selling_price * .06,
+                    }),
+                    Command.create({
+                        'name': 'Administrative fees',
+                        'quantity': '1',
+                        'price_unit': 100,
+                    })
+                ]
```

Comment:
Very good !
Doing it like this means the code is extendable by another module :D